end_to_end_testing/run.sh

```diff
@@ -5,8 +5,12 @@ set -e
 
 python --version
 
-chmod +x /usr/local/bin/reckoner
-reckoner version
+apt-get update
+apt-get install python3-pip
+
+cd /reckoner
+pip3 install reckoner
```

Comment:
This is installing the most recent version from pypi, I think we want to do a `pip3 install .` here, but I could be wrong.

Reply:
Good catch.
```suggestion
pip3 install .
```